spec/unit/application/lookup_spec.rb

```diff
@@ -145,6 +181,34 @@
       lookup.options[:explain] = true
       lookup.command_line.stubs(:args).returns(['a'])
       expect { lookup.run_command }.to output(<<-EXPLANATION).to_stdout
+Searching for 'lookup_options'
+  Merge strategy hash
+    Data Binding "hiera"
+      No such key: "lookup_options"
+    Data Provider "Hiera Data Provider, version 4"
+      ConfigurationPath "#{environmentpath}/production/hiera.yaml"
+      Data Provider "common"
+        Path "#{environmentpath}/production/data/common.yaml"
+          Original path: common
+          No such key: "lookup_options"
```

Comment:
This is after all for the benefit of the user - it seems beneficial to end a "Searching" with a conclusion of the result in all cases. The tree shows how the result was obtained.